### PR TITLE
Add wrapper for `PHPUnit_Framework_TestCase`

### PR DESCRIPTION
**Pull request by ZRazor**

### CODE REVIEW

File: Tests/Helpers/PHPUnit_Framework_TestCase.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ClassificationBundle\Tests\Helpers;
+
+/**
+ * This is helpers class for supporting old and new PHPUnit versions.
+ *
+ * @todo NEXT_MAJOR: Remove this class when dropping support for < PHPUnit 5.4.
```
Comment (OskarStark) [inline Tests/Helpers/PHPUnit_Framework_TestCase.php]: `@todo` needed @sonata-project/contributors ?

**Comment by jordisala1991** [comment]
If it is pedantic it does not require changelog right?

**Comment by greg0ire** [comment]
I removed it

**Comment by OskarStark** [comment]
> SonataDatagridBundle 2.2.1 which is used at the current moment doesn't have this wrapper, so I didnt apply this fix for the folowing files:

We should to the change then first on DatagridBundle, could you provide a PR and reference it?

**Comment by ZRazor** [comment]
@OskarStark, sure

**Comment by ZRazor** [comment]
Increase block-bundle dependency to ^3.3.1 to get `createMock` method inside `AbstractBlockServiceTestCase`.
https://github.com/sonata-project/SonataBlockBundle/pull/357

**Comment by OskarStark** [comment]
Thank you @ZRazor 👍

**Pull request merged**